px-map-behavior-layer-IMS.es6.js

```diff
@@ -162,6 +171,149 @@
       sketch: {
         type: Boolean,
         value: false
+      },
+
+      /**
+       * The current feature collection displayed by this layer.
+       * @type {Object}
+       */
+      featureCollection: {
+        type: Object
+      },
+
+      // Object to store feature id and feature pairs.
```

Comment:
Should have a matching JsDoc style comment as above.
Same for all newly added functions.